conda_libmamba_solver/solver.py

```diff
@@ -1004,7 +1005,20 @@ def _notify_conda_outdated(
             # This check can be silenced with a specific option in the context or in quiet mode
             return
 
-        current_conda_prefix_rec = PrefixData(context.conda_prefix).get("conda", None)
+        # manually check base prefix since `PrefixData(...).get("conda", None) is expensive
+        # once prefix data is lazy this might be a different situation
+        current_conda_prefix_rec = None
+        conda_meta_prefix_directory = os.path.join(context.conda_prefix, "conda-meta")
+        with suppress(OSError):
```

Comment:
```suggestion
        with suppress(OSError, ValueError):
```

`json.JSONDecodeError` is a subclass of `ValueError`, so we'll catch json decoding issues as well